adapter/src/components/LoginModal.js

```diff
@@ -11,10 +11,11 @@ import i18n from '../locales'
 import { post } from '../utils/api'
 
 const staticUrl = process.env.REACT_APP_DHIS2_BASE_URL
+const defaultUrl = process.env.REACT_APP_DHIS2_DEFAULT_BASE_URL
 
 export const LoginModal = () => {
     const [server, setServer] = useState(
-        staticUrl || window.localStorage.DHIS2_BASE_URL || ''
+        staticUrl || defaultUrl || window.localStorage.DHIS2_BASE_URL || ''
```

Comment:
I accidentally removed a comment by @mediremi here about that the `DHIS2_DEFAULT_BASE_URL` can be used for Netlify PR previews.

Reply:
Would be interesting to support a dropdown list of possible backend targets (through the same proxy) here - particularly useful for netlify deploys

Reply:
We could rename it to `DHIS2_DEFAULT_BASE_URLS` and make it a comma-delimited string of values? If multiple values are passed we render a dropdown otherwise we render an input field?

Reply:
Are the `LoginModal` changes required for the proxy to work or can we defer adding `REACT_APP_DHIS2_DEFAULT_BASE_URL` and `defaultUrl` until later ?

The Netlify PR deploys do not use the `start` command, they are static builds anyway, so the proxy won't help there right now.

Reply:
Same with this: https://github.com/dhis2/app-platform/pull/635/files#diff-bcd4b64d0aa7ede2599b25756aa2cb9c9f0e8000857f033edb963b85eb0f54e9R47 

Is that required for the proxy, or can it be deferred until we work on the Netlify deploy feature ?

Reply:
It's not required, it's just there to improve the dev UX by prefilling the login modal's server field to the proxy's URL.